run_pipe.py

```diff
@@ -407,12 +402,12 @@
                 #if (args.db_flag):
                 #    job_file.write("export PSRDB_TOKEN={0} \n".format(db_token))
                 #job_file.write("source /home/acameron/virtual-envs/meerpipe_db/bin/activate\n")
-                job_file.write("source /fred/oz005/pipelines/meerpipe/virtual-envs/meerpipe_db/bin/activate\n")
+                job_file.write(f"source {config_params['venv']}\n")
```

Comment:
Is the "f" before the source command required, or a typo?

Reply:
Required, it always you to use `{foo}` so you can add values to the string. This is easier to read than the `{}.format(foo)` method

Reply:
Ah, I see, so it's just a different way of writing the format command. Gotcha.